### ISSUE TITLE
Work with password

### ISSUE THREAD

**Issue by moonmeister** [emlun/yubikey-oath-dmenu#9]
This script doesn't seem to work when the password is set on my Yubikey. Is that expected or am I missing a config step?

Currently lights blink on the key if password is set but the menu never opens. It'd be nice if dmenu propted for password.

If I run this from a CLI, I get the following error:

```
Traceback (most recent call last):
  File "/usr/local/bin/yubikey-oath-dmenu", line 146, in <module>
    cli()
  File "/usr/lib/python3/dist-packages/click/core.py", line 759, in __call__
    return self.main(*args, **kwargs)
  File "/usr/lib/python3/dist-packages/click/core.py", line 714, in main
    rv = self.invoke(ctx)
  File "/usr/lib/python3/dist-packages/click/core.py", line 951, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/lib/python3/dist-packages/click/core.py", line 552, in invoke
    return callback(*args, **kwargs)
  File "/usr/lib/python3/dist-packages/click/decorators.py", line 17, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/usr/local/bin/yubikey-oath-dmenu", line 88, in cli
    for k, ctrl in controllers.items()
  File "/usr/local/bin/yubikey-oath-dmenu", line 88, in <dictcomp>
    for k, ctrl in controllers.items()
  File "/usr/lib/python3/dist-packages/ykman/oath.py", line 301, in list
    return list(_gen_creds())
  File "/usr/lib/python3/dist-packages/ykman/oath.py", line 293, in _gen_creds
    resp = self.send_apdu(INS.LIST, 0, 0)
  File "/usr/lib/python3/dist-packages/ykman/oath.py", line 259, in send_apdu
    raise APDUError(resp, sw)
ykman.driver_ccid.APDUError: APDU error: SW=0x6982
```

Thanks for making this available and for your assistance!!

**Comment by emlun** [comment]
Thanks! I hadn't thought of that, so this isn't supported. It shouldn't be hard to do, though. I'll take a look at it.

**Comment by emlun** [comment]
Released in 0.8.0! Thanks again!
